atlasdb-processors-tests/build.gradle

```diff
@@ -0,0 +1,12 @@
+apply from: "../gradle/publish-jars.gradle"
+apply from: "../gradle/shared.gradle"
+
+apply plugin: 'org.inferred.processors'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    processor project(":atlasdb-processors")
```

Comment:
feels a little odd to have a separate project for tests. why can't this be in `src/test/java` in the main project?